### DISCUSSION TITLE
tags

### DISCUSSION THREAD

**Discussion by bmason** [fabricjs/fabric.js#7335] [Q&A]
I want to use tags in the text area.  I like the functionality of [Tagify](https://github.com/yairEO/tagify) in mixed mode.

My first attempt was:

`canvas.on('mouse:dblclick', function(options) {
	new Tagify(document.querySelector("[data-fabric-hiddentextarea]"),this.tagifySettings);
  });`

This works fine on an input field on the same page.  I believe I targeted the right element at the right time but I am not seeing the Tagify functionality.  Tracing through the events is tedious, so I am hoping someone has some experience with this.

**Comment by asturur** [comment]
This is not going to work right away.
The fabricTextbox is not meant for text input in that sense, is just to edit a graphical text.
This is a functionality that belong to a UI, use case that fabricJS can't support.

I think doing this inside the fabric code is hard, while possible, you should really consider how much is important for you that feature on the canvas.
